packages/mux-player/src/index.ts

```diff
@@ -318,6 +339,33 @@ class MuxPlayerElement extends VideoApiElement implements MuxPlayerElement {
     render(template(getProps(this, { ...this.#state, ...props })), this.shadowRoot as Node);
   }
 
+  #setUpThemeAttributes() {
+    // Forward `theme-` prefixed attributes to the theme.
+    // e.g. `theme-control-bar-vertical` for the Micro theme.
+    const setThemeAttribute = (attributeName: string | null) => {
+      if (!attributeName?.startsWith('theme-')) return;
+
+      const themeAttrName = attributeName.replace(/^theme-/, '');
+      if (ThemeAttributeNames.includes(themeAttrName)) return;
+
+      const value = this.getAttribute(attributeName);
+      if (value != null) {
+        this.mediaTheme?.setAttribute(themeAttrName, value);
+      } else {
+        this.mediaTheme?.removeAttribute(themeAttrName);
+      }
+    };
+
+    const observer = new MutationObserver((mutationList) => {
```

Comment:
should we store this observer in a private var so that we can disconnect it on `disconnectedCallback`?

Reply:
briefly discussed internally, opt to keep the attribute forwarding working even if disconnected for now.
if the element is released from memory, the observer should get cleaned up 
https://github.com/WICG/webcomponents/issues/550#issuecomment-460015529